frontend/src/components/Product/ProductRankingList/ProductRankingList.tsx

```diff
@@ -7,7 +7,7 @@ const ProductRankingList = () => {
     <ul>
       {productRanking.map((productRanking) => (
         <li key={productRanking.id}>
-          <ProductRankingItem productRanking={productRanking} />
+          <ProductRankingItem productRanking={productRanking} variant="ranking" />
```

Comment:
```suggestion
          <ProductOverviewItem rank={productRanking.rank} name={productRanking.name} image={productRanking.image} />
```

그렇다면 여기는 이런식으로 될 수 있겠네요. rank값은 API가 어떻게 올지 모르지만 index를 활용할 수 있겠네요.

랭킹이 없는 상품 요약의 경우는 
```
<ProductOverviewItem name={productRanking.name} image={productRanking.image} />
```
요런 느낌일거에요.